pkg/controller/controller_instance.go

```diff
@@ -747,19 +754,20 @@ func (c *controller) processServiceInstancePollingFailureRetryTimeout(instance *
 
 // resolveReferences checks to see if ClusterServiceClassRef and/or ClusterServicePlanRef are
 // nil and if so, will resolve the references and update the instance.
+// If references don't need to be resolved, the method returns false as its 2nd return value
```

Comment:
I think this comment is a bit misleading.
For example, we return `false` when `ClusterServiceClassRef` is missing, but there is an error resolving it.
Can we rephrase it to "If references needed to be resolved, and the instance status was successfully updated, the method returns 'true'"?